### DISCUSSION TITLE
How to use the S3 deprecated path style on a HTTP endpoint?

### DISCUSSION THREAD

**Discussion by fungiboletus** [iterate-ch/cyberduck#14388] [Q&A] [answered]
I have a S3 server, using Minio, that is forwarded to my local computer.

It doesn't have encryption, and I can list the buckets using http://localhost:9000 (assuming it is forwarded on port 9000). However I cannot list a bucket, as I get the following error:

```
Failed to parse XML document with handler class org.jets3t.service.impl.rest.XmlResponsesSaxParser$ListBucketHandler. Please contact your web hosting service provider for assistance.
```

I am guessing that it is related to the deprecated path style. However I don't find how to configure cyberduck to connect to a S3 endpoint without TLS, and without the non deprecated path style.

**Comment by dkocher** [comment, marked as answer]
Use the _S3 (Deprecated path style requests) connection profile_ [^1]. All connection profiles are available through the _Preferences → Profiles_ tab.

[^1]: https://docs.cyberduck.io/protocols/s3/#disable-use-of-virtual-host-style-requests

**Reply by fungiboletus** [reply]
This profile doesn't support HTTP.

**Reply by dkocher** [reply]
You are right. While we have a S3 (HTTP) profile there is none for path-style requests.

**Reply by AliveDevil** [reply]
Added missing profile now, you can either download it "Raw" from Github, or wait until it is merged, then select it from the Profiles-preferences tab.

https://github.com/iterate-ch/profiles/pull/34

**Reply by fungiboletus** [reply]
Thanks for the fast change! I downloaded the file and it works well.
